public/styles/style.css

```diff
@@ -32,12 +35,14 @@
 }
 
 @media screen and (max-width: 768px) {
+    #character .table-custom>article,
     #favorites .table-custom>article {
         flex-wrap: wrap;
     }
     #favorites .table-name {
         flex-basis: 6rem;
     }
+    #character .table-count,
```

Comment:
This is a mistake I think. This should be deleted, and on line 50, add #character .table quote, to change the order